src/traces/sankey/render.js

```diff
@@ -444,11 +444,11 @@ module.exports = function(svg, styledData, layout, callbacks) {
         .append('g')
         .classed('sankeyLinks', true)
         .style('fill', 'none')
-        .style('transform', linksTransform);
```

Comment:
We need to check if the `style`d element is an SVG element or not. (I'm planning to just check the first element in the selection with `sel.node()` as selecting both HTML and SVG elements in the same selection is not commonly done.) A linting problem arises:
```
/Users/robert/repos/plotly/plotly.js/test/image/strict-d3.js
  42:38  error  'SVGElement' is not defined  no-undef
```
Interestingly the linter doesn't balk on a [similar usage](https://github.com/plotly/plotly.js/blob/master/src/lib/index.js#L426).

Have we got an established workaround, or should I just lint-define the `SVGElement` locally?